cmd/varlogsn/varlogsn.go

```diff
@@ -134,12 +134,17 @@ func start(c *cli.Context) error {
 	if err != nil {
 		return err
 	}
+	memTableSize, err := units.FromByteSizeString(c.String(flagStorageMemTableSize.Name))
+	if err != nil {
+		return err
+	}
+
 	storageOpts := []storage.Option{
 		storage.WithL0CompactionThreshold(c.Int(flagStorageL0CompactionThreshold.Name)),
 		storage.WithL0StopWritesThreshold(c.Int(flagStorageL0StopWritesThreshold.Name)),
 		storage.WithLBaseMaxBytes(lbaseMaxBytes),
 		storage.WithMaxOpenFiles(c.Int(flagStorageMaxOpenFiles.Name)),
-		storage.WithMemTableSize(c.Int(flagStorageMemTableSize.Name)),
+		storage.WithMemTableSize(int(memTableSize)),
```

Comment:
I don't understand why the old way didn't work. Is `flagStorageMaxOpenFiles` applied?

Reply:
The flag `flagStorageMemTableSize` is a string, for instance, "32M", "1G" and so on. However, `c.Int(flagStorageMemTableSize.Name)` parses "32M" into "32" not "32<<20".

Reply:
As far as I know, the flag `flagStorageMaxOpenFiles` works well because it parses the flag value as an integer.